### PR TITLE
Fix #13464: Changed default time stretch property for fermatas

### LINKED ISSUE #13464: [MU4 Issue] Change default time stretch property for fermatas
**Describe the bug**
Fermatas of all varieties currently play with a default "time stretch" property of 100%.
This value produces no discernible audible change, giving the impression that the fermata is not being recognised in playback. 

**To Reproduce**
See video demo: 

https://user-images.githubusercontent.com/86290556/191978684-e6add74e-0763-4032-a0a9-bbe539dc154f.mov

**Expected behavior**
@oktophonie and I have consulted together on making the following recommendations for default time stretch values: 

| Fermata type | Recommended time stretch value |
| ------------- | ----------------------------------|
| Very short fermata above | 125% |
| Short fermata above | 150% |
| Short fermata (Henze) above | 150% |
| Fermata above | 200% |
| Long fermata above | 300% |
| Long fermata (Henze) above | 300% |
| Very long fermata above | 400% |

**Platform information**
 - MacOS 12.3.1 (M1)

### PR DESCRIPTION
**Pull request by HemantAntony**
Resolves: #13464 

- [x] I signed [CLA](https://musescore.org/en/cla)
- [x] I made sure the code in the PR follows [the coding rules](https://github.com/musescore/MuseScore/wiki/CodeGuidelines)
- [x] I made sure the code compiles on my machine
- [x] I made sure there are no unnecessary changes in the code
- [x] I made sure the title of the PR reflects the core meaning of the issue you are solving
- [x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"
- [x] I made sure the commit message title starts with "fix #424242:" if there is a related issue
- [ ] I created the test (mtest, vtest, script test) to verify the changes I made

### CODE REVIEW

**Comment by HemantAntony** [comment]
Hope this is a good fix 😉

**Comment by bkunda** [comment]
Thanks so much (again!) @HemantAntony 
🟢 Tested on Win and MacOS. Fixed.
Good to merge from my end.

File: src/engraving/libmscore/fermata.h [hunk 1]
```diff
@@ -63,7 +63,7 @@ class Fermata final : public EngravingItem
     double mag() const override;
 
     SymId symId() const { return _symId; }
-    void setSymId(SymId id) { _symId  = id; }
+    void setSymId(SymId id) { _symId  = id; _timeStretch = _timeStretch == -1 ? propertyDefault(Pid::TIME_STRETCH).value<double>() : -1; }
```
Comment (cbjeukendrup) [inline src/engraving/libmscore/fermata.h:66]: Now that this getter is not so trivial anymore, let's move it to .cpp

File: src/engraving/libmscore/fermata.h [hunk 2]
```diff
@@ -44,7 +44,7 @@ class Fermata final : public EngravingItem
     OBJECT_ALLOCATOR(engraving, Fermata)
 
     SymId _symId;
-    double _timeStretch;
+    double _timeStretch = -1;
```
Comment (cbjeukendrup) [inline src/engraving/libmscore/fermata.h:47]: Better initialise as `-1.0`

Comment (HemantAntony) [reply src/engraving/libmscore/fermata.h:47]: Okay, just asking is there a specific reason for doing so. Or is it just for looks?

Comment (cbjeukendrup) [reply src/engraving/libmscore/fermata.h:47]: Apart from that it is slightly clearer, it eliminates an implicit conversion from int to double.

Comment (HemantAntony) [reply src/engraving/libmscore/fermata.h:47]: Oh okay

**Review by RomanPudashkin — APPROVED** [review]

**Comment by RomanPudashkin** [comment]
@HemantAntony please rebase it

**Comment by HemantAntony** [comment]
@RomanPudashkin done

**Pull request merged**